<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabRankListWidget</class>
 <widget class="QWidget" name="TabRankListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="grpBoxBoard">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>671</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>榜单-前三名</string>
   </property>
   <widget class="QLabel" name="labelServer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>服务器</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineA0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>000.000.000.000</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineB0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>000.000.000.000</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineC0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>000.000.000.000</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelMaster">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>主账号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineC1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>a368</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineA1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>a168</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineB1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>a268</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineC2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>110</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>000.000.000.000</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelClient">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>客户端</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineB2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>000.000.000.000</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineA2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>000.000.000.000</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelOpUser">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>操作用户</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineA3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>aa1</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineC3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>110</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>cc3</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineB3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>80</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>bb2</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineC4">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>110</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>别名验证</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineA4">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>50</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>获取点数</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelOpType">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>操作类型</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineB4">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>80</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>用户名验证</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grpBoxRankCond">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>761</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>排行条件</string>
   </property>
   <widget class="QLabel" name="labelA0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>排行信息</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxRankInfo">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>服务器</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>主账号</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>客户端</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>操作用户</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>操作类型</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>错误信息</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelA2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>结束时间:</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateTimeTo">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>yyyy-MM-dd hh:mm:ss</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateTimeFrom">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>yyyy-MM-dd hh:mm:ss</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelA1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>开始时间:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelA3">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>操作用户:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineOpUser">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>30</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>空表示全部</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grpBoxOp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>681</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>操作</string>
   </property>
   <widget class="QPushButton" name="btnReset">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>重置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRank">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>排行</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClearBoard">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>清空榜单</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClearRankList">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>清空排行榜</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grpBoxRankList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>631</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>排行榜</string>
   </property>
   <widget class="QTableWidget" name="tableRankList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>581</width>
      <height>101</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>排行信息</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>数量</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>总数 </string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>百分比</string>
     </property>
    </column>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>comboBoxRankInfo</tabstop>
  <tabstop>dateTimeFrom</tabstop>
  <tabstop>dateTimeTo</tabstop>
  <tabstop>lineOpUser</tabstop>
  <tabstop>btnRank</tabstop>
  <tabstop>btnReset</tabstop>
  <tabstop>btnClearBoard</tabstop>
  <tabstop>btnClearRankList</tabstop>
  <tabstop>lineA0</tabstop>
  <tabstop>lineB0</tabstop>
  <tabstop>lineC0</tabstop>
  <tabstop>lineA1</tabstop>
  <tabstop>lineB1</tabstop>
  <tabstop>lineC1</tabstop>
  <tabstop>lineA2</tabstop>
  <tabstop>lineB2</tabstop>
  <tabstop>lineC2</tabstop>
  <tabstop>lineA3</tabstop>
  <tabstop>lineB3</tabstop>
  <tabstop>lineC3</tabstop>
  <tabstop>lineA4</tabstop>
  <tabstop>lineB4</tabstop>
  <tabstop>lineC4</tabstop>
  <tabstop>tableRankList</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
